gravitee-apim-portal-webui-next/src/components/subscription/webhook/consumer-configuration-authentification/consumer-configuration-authentication.component.html

```diff
@@ -0,0 +1,113 @@
+<!--
+
+    Copyright (C) 2025 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<app-accordion>
+  <app-accordion-title>
+    <span class="m3-title-medium" i18n="@@subscriptionConfigurationAuthentication">Security configuration</span>
+  </app-accordion-title>
+  <form class="form" [formGroup]="authForm">
+    <mat-form-field class="form__field type">
+      <mat-label i18n="@@subscriptionConfigurationAuthenticationType">Type</mat-label>
+      <mat-select formControlName="type" id="authType">
+        @for (type of types; track type.value) {
+          <mat-option [value]="type.value">{{ type.label }}</mat-option>
+        }
+      </mat-select>
+    </mat-form-field>
+
+    @if (authForm.controls.type.value !== 'none') {
+      <div>
+        @switch (authForm.controls.type.value) {
+          @case ('basic') {
+            <mat-form-field class="form__field">
+              <mat-label i18n="@@subscriptionConfigurationAuthenticationUsername">Username</mat-label>
+              <input matInput formControlName="username" name="username" required />
+              <mat-hint i18n="@@subscriptionConfigurationAuthenticationUsernameHint">The user name</mat-hint>
+              @if (authForm.controls.username.hasError('required')) {
+                <mat-error i18n="@@subscriptionConfigurationAuthenticationUsernameRequired">Username is required</mat-error>
+              }
+            </mat-form-field>
+
+            <mat-form-field class="form__field">
+              <mat-label i18n="@@subscriptionConfigurationAuthenticationPassword">Password</mat-label>
+              <input matInput formControlName="password" type="password" name="password" autocomplete="off" required />
+              <mat-hint i18n="@@subscriptionConfigurationAuthenticationPasswordHint">The user password</mat-hint>
```

Comment:
Same thing, I'm not sure if the hint is necessary. WDYT?

It if you keep it, I would suggest:
```suggestion
              <mat-hint i18n="@@subscriptionConfigurationAuthenticationPasswordHint">The user's password</mat-hint>
```